src/content/caseStudies/codetv.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Unleashing creativity'
+description: 'How CodeTV built a video streaming platform with Astro and Mux'
+seo:
+  title: 'Unleashing creativity: How CodeTV built a video streaming platform with Astro and Mux'
+  description: 'How Astro gives CodeTV everything they need to run a subscription-based TV service.'
+coverGradientFrom: '#3245FF'
+coverGradientTo: '#BC52EE'
+publishDate: 'September 9, 2025'
+authors:
+  - astro-team
+lang: 'en'
+socialImage: /src/content/caseStudies/_images/codetv/social-image.webp
+coverImage: /src/content/caseStudies/_images/codetv/cover-image.webp
+headerImage: /src/content/caseStudies/_images/codetv/header.png
```

Comment:
```suggestion
headerImage: /src/content/caseStudies/_images/codetv/header.png
related:
  - mux-official-video-partner
```
This might break the build without the other post being published yet, and haven't confirmed that a case study can take this, so do not commit this yet!
```